lib/libqrtr.h

```diff
@@ -167,11 +167,23 @@ ssize_t qmi_encode_message(struct qrtr_packet *pkt, int type, int msg_id,
 			   int txn_id, const void *c_struct,
 			   struct qmi_elem_info *ei);
 
+struct qmi_tlv {
+	void *allocated;
+	void *buf;
+	size_t size;
+	int error;
+};
+
+struct qmi_tlv_item {
+	uint8_t key;
+	uint16_t len;
+	uint8_t data[];
+} __attribute__((__packed__));
+
 struct qmi_tlv *qmi_tlv_init(uint16_t txn, uint32_t msg_id, uint32_t msg_type);
 void *qmi_tlv_encode(struct qmi_tlv *tlv, size_t *len);
 struct qmi_tlv *qmi_tlv_decode(void *buf, size_t len);
 void qmi_tlv_free(struct qmi_tlv *tlv);
-void qmi_tlv_dump(struct qmi_tlv *tlv);
```

Comment:
Oh, my. I started to like the function and now it is gone. Squash everything together so that the remnants of it won't make me mourn the loss each time I look at the git history.